### PR TITLE
Character Order sort Highest to Lowest

### LINKED ISSUE #5: Character Order on Dashboard
# Acceptance Criteria
As a user viewing the Dashboard, the order of Characters displays highest Initiative to lowest.

### PR DESCRIPTION
**Pull request by JonnoKiwi**
Closes https://github.com/JonnoKiwi/initiative-tree/issues/5

### CODE REVIEW

File: app/screens/Dashboard/CharactersRolls.tsx [hunk 1]
```diff
@@ -47,6 +47,8 @@ export default function CharactersRolls (props) {
     onEditCharacter
   } = props
   const list = data || []
+  list.sort((a, b) => (a.initiative < b.initiative) ? 1 : -1)
```
Comment (johntimothybailey) [inline app/screens/Dashboard/CharactersRolls.tsx]: This must be done in the State Machine specifically in the api.fake.ts file for the Characters. you don't want business / state logic done in your presentation layer

**Review by johntimothybailey — CHANGES_REQUESTED** [review]

File: app/screens/Dashboard/CharactersRolls.tsx (hunk 1 above)
Comment (JonnoKiwi) [reply app/screens/Dashboard/CharactersRolls.tsx]: Ok, I'll move it out.

File: app/services/api/characters/api.fake.ts [hunk 2]
```diff
@@ -5,7 +5,10 @@ import { ApiConfig, HydrogenAPI, SagaSauceAPI } from '../IHydrogenAPI'
 import R from 'ramda'
 import AsyncStorage from '@react-native-async-storage/async-storage'
 
-const sortBy = R.sortBy(R.prop('initiative'))
+const sortWith = R.sortWith([
```
Comment (johntimothybailey) [inline app/services/api/characters/api.fake.ts:8]: @JonnoKiwi WELL DONE!!!!!! Great job researching this and finding it!

**Review by johntimothybailey — APPROVED** [review]

File: app/services/api/characters/api.fake.ts (hunk 2 above)
Comment (JonnoKiwi) [reply app/services/api/characters/api.fake.ts:8]: Thanks John, it's satisfying when it comes together  ^^

**Pull request merged**